.github/workflows/pyopenms-wheels.yml

```diff
@@ -188,26 +238,36 @@ jobs:
           brew install --quiet doxygen ghostscript graphviz
         fi
 
-    - name: Build
+    - name: Build on macos arm
       run: |
         PYTHON_VERSIONS=$(cat OpenMS/.github/workflows/python_versions.json)
-        export CC=clang
-        export CXX=clang++
-        export MACOSX_DEPLOYMENT_TARGET=12
+        # export CC=clang
+        # export CXX=clang++
+        # export MACOSX_DEPLOYMENT_TARGET=12
         # Unfortunately, on macOS due to the inofficial way of enabling OpenMP on AppleClang, passing the following
         # options to setup.py extra_compile_args does not work. See also https://gist.github.com/andyfaff/084005bee32aee83d6b59e843278ab3e
         export CFLAGS="-Xpreprocessor -fopenmp $CFLAGS"
         export CXXFLAGS="-Xpreprocessor -fopenmp $CXXFLAGS"
 
-        mkdir bld
-        pushd bld
+        # mkdir bld
+        # pushd bld
         # Use -DCMAKE_FIND_DEBUG_MODE=ON for debug
-        cmake -DCMAKE_BUILD_TYPE="Release" -DCMAKE_PREFIX_PATH="$(brew --prefix libomp);$(echo $Qt5_Dir)/lib/cmake;${Qt5_Dir}" -DCMAKE_OSX_DEPLOYMENT_TARGET=12 -DBOOST_USE_STATIC=OFF ../OpenMS
-        make -j${{ steps.cpu-cores.outputs.count }} OpenMS
+
+        export CMAKE_PREFIX_PATH="$(brew --prefix libomp);$(echo $Qt5_Dir)/lib/cmake;${Qt5_Dir}"
```

Comment:
should be Qt5_DIR

Reply:
but since it seems to have found Qt before, this is probably not the main issue.

Reply:
still, you should try, maybe the empty entry confuses cmake.
you could also print the `tree` output of the libomp folder, to make sure everything is there.

Reply:
```suggestion
        export CMAKE_PREFIX_PATH="$(brew --prefix libomp);$(echo $Qt5_DIR)/lib/cmake;${Qt5_DIR}"
```

Reply:
Nope, looks like it still has the same behavior:
-- Looking for setuptools - found setuptools 75.1.0, version ok
-- Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES) 
-- Could NOT find OpenMP (missing: OpenMP_CXX_FOUND CXX) 
-- Py extra args